### PR TITLE
Removed the legacy client from the website

### PR DESCRIPTION
**Pull request by louvenarde**
Download client is now a direct link to the latest Java client release, the client comparison page is removed.

### CODE REVIEW

File: templates/views/index.pug [hunk 1]
```diff
@@ -9,6 +9,10 @@ block content
 				h1 Forged Alliance Forever
 				h3 The Community Driven Lobby for </br> Supreme Commander: Forged Alliance
 			a(class="btn btn-default btn-lg btn-outro btn-danger", href='/client') Download Client
+			p 
+				| Any issue ?
```
Comment (germanicianus) [inline templates/views/index.pug]: Remove blank in front of ‚?‘.

File: templates/views/index.pug [hunk 2]
```diff
@@ -9,6 +9,10 @@ block content
 				h1 Forged Alliance Forever
 				h3 The Community Driven Lobby for </br> Supreme Commander: Forged Alliance
 			a(class="btn btn-default btn-lg btn-outro btn-danger", href='/client') Download Client
+			p 
+				| Any issue ? 
+				a(href=client_download_link, style="color:white;text-decoration:underline;") Try the legacy client
```
Comment (germanicianus) [inline templates/views/index.pug]: Only link ‚legacy client‘, add ‚.‘ at end of sentence.

File: templates/views/index.pug (hunk 1 above)
Comment (louvenarde) [reply templates/views/index.pug]: sorry, french punctuation :P

File: templates/views/index.pug (hunk 2 above)
Comment (louvenarde) [reply templates/views/index.pug]: I don't think it's necessary to add a dot, but i'm going to fix the link

Comment (germanicianus) [reply templates/views/index.pug]: Sentences always need punctuation. Don’t you think so?

Comment (germanicianus) [reply templates/views/index.pug]: But don’t include the dot in the link.

Comment (louvenarde) [reply templates/views/index.pug]: Well no not in that case, the sentence is not part of a paragraph, it's floating around, there's no need to end it with a dot if nothing of the same weight is coming after it
Just like there is no dot on the orange "Download client" button

**Pull request merged**